### PR TITLE
Landing pages backend scaffold

### PR DESCRIPTION
**Pull request by KludgeKML**
⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

## What

Add the scaffolding required to render flexible "blocks", where the type of block, and the order of the blocks can be controlled by the user.

## Why

A new flexible content type called a landing page is being created.
The content type will allow publishers to pick the "blocks" that should be included on the page, e.g. govspeak, tabs, image, video, and decide on the order of these "blocks".

## How

The publishing app doesn't exist for these blocks yet, nor does a content schema. 

Therefore, for the time being dummy content is being hard-coded. This will be replaced by actual hard-coded content when the first landing page is ready to go live. The hard-coded content will be removed when it is possible to publish the content from a CMS.

As a content schema doesn't exist for the landing_page type yet, a fake content item has been added to the controller. This will be removed in a later PR once a content schema exists.

## Screenshots?
[Rendered page](https://govuk-frontend-app-pr-4247.herokuapp.com/landing-page)

### CODE REVIEW

File: lib/data/landing_page_content_items/landing_page.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+blocks:
+- type: paragraph
+  content: Here's some content!
```
Comment (richardTowers) [inline lib/data/landing_page_content_items/landing_page.yaml:3]: I was thinking we could use the [govspeak component](https://components.publishing.service.gov.uk/component-guide/govspeak) for things that are a bit long-form-text-like. That way we can keep our blocks to high level constructs, and avoid reproducing paragraphs / lists / tables etc.

Comment (richardTowers) [reply lib/data/landing_page_content_items/landing_page.yaml:3]: Gives the user a lot of flexibility (possibly too much, but I think it's better to err on that side of things for this project)

Comment (KludgeKML) [reply lib/data/landing_page_content_items/landing_page.yaml:3]: Yeah, that makes sense. The paragraph was just for the proof of concept, I'll replace it with a govspeak block.

Comment (KludgeKML) [reply lib/data/landing_page_content_items/landing_page.yaml]: Done

Comment (KludgeKML) [reply lib/data/landing_page_content_items/landing_page.yaml]: Only remaining question here is for this version, do we allow govspeak in the content for that item or do we stick closer to what we'll get in the end from content store (HTML fragments converted from govspeak, which is what the component takes). I've chosen the latter path for now, I assume we generally do not want to be converting govspeak to HTML in the frontend.

Comment (richardTowers) [reply lib/data/landing_page_content_items/landing_page.yaml]: Hmm, that's a good question. It's a bit annoying to have to hardcode it as HTML, but I think that'll do for now. Options for later:

- Hardcode as HTML (current version)
- Hardcode as govspeak, have a build step to swap it out for HTML
- Use a real content item (publishing-api will render the govspeak)

**Review by richardTowers — APPROVED** [review]
Nice work both! Looks good to me!

**Pull request merged**